src/main/java/com/parsing/parsers/parserlot/ParserLotService.java

```diff
@@ -0,0 +1,118 @@
+package com.parsing.parsers.parserlot;
+
+import com.parsing.model.LotResult;
+import com.parsing.model.Status;
+import com.parsing.repository.LotResultRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+@RequiredArgsConstructor
+@Slf4j
+@Service
+@Transactional(readOnly = true)
+public class ParserLotService {
+
+    private final LotResultRepository lotRepository;
+    List<String> urls = prepareURI();
```

Comment:
Do we really need a state for the Service? Services should be stateless so you need to have only constants here or fields with @Value annotation but not some container.

Reply:
I changed this list to "private static final". And we will escape from the state in this situation, I hope.